From 2726b2f10d590425aa1c71d19c6aade1333b8c77 Mon Sep 17 00:00:00 2001
From: Yi Zhao <yi.zhao@windriver.com>
Date: Sun, 8 Oct 2023 13:06:00 +0800
Subject: [PATCH] getent.py/chsh.py: switch to python3

There is no python symblic link by default on system which will cause
runtime error for getent.ldap and chsh.ldap commands:
$ getent.ldap
/usr/bin/env: 'python': No such file or directory

Switch to python3 for these commands.

Upstream-Status: Pending

Signed-off-by: Yi Zhao <yi.zhao@windriver.com>
---
 utils/chsh.py   | 4 ++--
 utils/getent.py | 2 +-
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/utils/chsh.py b/utils/chsh.py
index c48459d..9cbec49 100755
--- a/utils/chsh.py
+++ b/utils/chsh.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 # coding: utf-8
 
 # chsh.py - program for changing the login shell using nslcd
@@ -42,7 +42,7 @@ parser.add_argument('username', metavar='USER', nargs='?',
 
 def ask_shell(oldshell):
     """Ask the user to provide a shell."""
-    shell = raw_input('  Login Shell [%s]: ' % oldshell)
+    shell = input('  Login Shell [%s]: ' % oldshell)
     return shell or oldshell
 
 
diff --git a/utils/getent.py b/utils/getent.py
index ae00c5e..a958245 100755
--- a/utils/getent.py
+++ b/utils/getent.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 # coding: utf-8
 
 # getent.py - program for querying nslcd
-- 
2.25.1

